packages/generator/ni_measurement_plugin_sdk_generator/ni_measurement_plugin_client_generator/_constants.py

```diff
@@ -0,0 +1,36 @@
+"""Measurement plug-in client generator constants."""
+
+from google.protobuf.type_pb2 import Field
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import (
+    DoubleXYData,
+)
+
+_V2_MEASUREMENT_SERVICE_INTERFACE = "ni.measurementlink.measurement.v2.MeasurementService"
+
+_INVALID_CHARS = "`~!@#$%^&*()-+={}[]\|:;',<>.?/ \n"
+
+_XY_DATA_IMPORT = "from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import DoubleXYData"
+_PATH_IMPORT = "from pathlib import Path"
+
+_DEFAULT_IMPORT_TYPE = "In-Built"
+_CUSTOM_IMPORT_TYPE = "Custom"
+_IMPORT_MODULES: dict[str, str] = {}
```

Comment:
Python 3.8 doesn't support subscripting built-in types. Use `typing.Dict[str, str]`, `typing.List[T]`, etc.

```
Run poetry run pytest -v --cov=ni_measurement_plugin_sdk_generator --junitxml=test_results/nimg-windows-latest-py3.8.xml
============================= test session starts =============================
platform win32 -- Python 3.8.10, pytest-8.3.2, pluggy-1.5.0 -- D:\a\measurement-plugin-python\measurement-plugin-python\packages\generator\.venv\Scripts\python.exe
cachedir: .pytest_cache
rootdir: D:\a\measurement-plugin-python\measurement-plugin-python\packages\generator
configfile: pyproject.toml
testpaths: tests
plugins: cov-5.0.0
collecting ... collected 2 items / 2 errors

=================================== ERRORS ====================================
_________ ERROR collecting tests/acceptance/test_client_generator.py __________
tests\acceptance\test_client_generator.py:10: in <module>
    from ni_measurement_plugin_sdk_generator.ni_measurement_plugin_client_generator import template
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\template.py:14: in <module>
    from ni_measurement_plugin_sdk_generator.ni_measurement_plugin_client_generator._constants import (
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\_constants.py:17: in <module>
    _IMPORT_MODULES: dict[str, str] = {}
E   TypeError: 'type' object is not subscriptable
_________ ERROR collecting tests/acceptance/test_client_generator.py __________
tests\acceptance\test_client_generator.py:10: in <module>
    from ni_measurement_plugin_sdk_generator.ni_measurement_plugin_client_generator import template
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\template.py:14: in <module>
    from ni_measurement_plugin_sdk_generator.ni_measurement_plugin_client_generator._constants import (
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\_constants.py:17: in <module>
    _IMPORT_MODULES: dict[str, str] = {}
E   TypeError: 'type' object is not subscriptable
============================== warnings summary ===============================
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\_constants.py:10
  D:\a\measurement-plugin-python\measurement-plugin-python\packages\generator\ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\_constants.py:10: DeprecationWarning: invalid escape sequence \|
    _INVALID_CHARS = "`~!@#$%^&*()-+={}[]\|:;',<>.?/ \n"

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
- generated xml file: D:\a\measurement-plugin-python\measurement-plugin-python\packages\generator\test_results\nimg-windows-latest-py3.8.xml -

---------- coverage: platform win32, python 3.8.10-final-0 -----------
Name                                                                                       Stmts   Miss  Cover
--------------------------------------------------------------------------------------------------------------
ni_measurement_plugin_sdk_generator\__init__.py                                                0      0   100%
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\_constants.py      10      1    90%
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_client_generator\template.py        47     39    17%
ni_measurement_plugin_sdk_generator\ni_measurement_plugin_generator\template.py               79     52    34%
--------------------------------------------------------------------------------------------------------------
TOTAL                                                                                        [13](https://github.com/ni/measurement-plugin-python/actions/runs/10388050708/job/28762900713?pr=841#step:16:14)6     92    32%

=========================== short test summary info ===========================
ERROR tests/acceptance/test_client_generator.py - TypeError: 'type' object is not subscriptable
ERROR tests/acceptance/test_client_generator.py - TypeError: 'type' object is not subscriptable
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!
======================== 1 warning, 2 errors in 0.72s =========================
Error: Process completed with exit code 1.
```